### DISCUSSION TITLE
Hacked<title>

### DISCUSSION THREAD

**Discussion by Puffylow** [community/community#59831] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Im a single mom 2 underage girls. I was being watched by someone using this coding program.  Im a private resident. He took pictures.  How can I find him and was this created for watching private homes? Please help. They will not stop. It's everyday. I've changed phones, sim cards, and networks. My daughter's can not come to my home because it is not safe.  I saw a username Astronaut. Please please help!!!

**Comment by leonden** [comment]
If I understand your problem correctly, I would immediately seek help at your local police station.
What do you mean exactly with:
> I was being watched by someone using this coding program.

**Comment by samus-aran** [comment]
I’m sorry that you’re having to go through this, @Puffylow  

It sounds like you have been a victim of serious harassment and/or abuse. Though we can’t directly help with your situation, you might be able to receive assistance through [Crash Override Network](http://www.crashoverridenetwork.com/) and [RAINN](https://www.rainn.org/).
